catalog-spec/catalog-spec.md

```diff
@@ -53,9 +53,9 @@ with links.
 
 A more complete list of possible 'rel' types can be seen at the [IANA page of Link Relation Types](https://www.iana.org/assignments/link-relations/link-relations.xhtml).
 
-Please see the chapter 'relative vs absolute links' in the [Item spec](../item-spec/item-spec.md#relative-vs-absolute-links)
-for a discussion on that topic, as well as the [use of links](../best-practices.md#use-of-links) section of the 
-catalog best practices document.
+Currently, the JSON schema for links does not require them to be formatted as URIs, to allow implementors 
```

Comment:
Hmm, I'm actually not sure whether that is true. I need to check...

Reply:
JSON Schema got a new format called "uri-reference" with draft 6. A URI Reference is either a URI or a relative-reference, according to RFC3986, section 4.1. So I think we should just add this to the JSON Schemas for links, urls, assets etc. Then we can adopt the language here. I'll put that on my todo.

Reply:
I've implemented the "iri-reference" format, see commit https://github.com/radiantearth/stac-spec/pull/953/commits/2f223ae402647308bfadb258354ec55c997ec315 - This may require some additional effort in tooling to validate it correctly as iri-reference may not be directly supported. STAC Node Validator for example can't validate it by default, see https://github.com/m-mohr/stac-node-validator/issues/6

I won't approve this PR myself as I've contributed this change and would love if at least two other implementors of validation tools (looking at @lossyrob and @jbants) could check whether this would lead to major issues.